concent_api/concent_api/settings/production.py

```diff
@@ -2,3 +2,6 @@
 from .base import *  # NOQA  # pylint: disable=wildcard-import
 
 DEBUG = False
+
+CONCENT_PUBLIC_KEY  = b'D\x1e\x9d\xfe\x0b3\x1a\x9e\xaf\xe8\x1d^\xbb\xf1~\xb1\x15A\xce\xda\xf5\x8f\xe4\x166\x9b\xa9\x00\xe1+]\xcd\xe7\x7f\x93\x17t\xa6\xbd\xbfY\xcf|]\xa5\x9e;|lJ\xbf(q\xb7`\x12S\xcd\n5\xdey\x86\xa5'
+CONCENT_PRIVATE_KEY = b'[\x10!\xca\rv\xb1\x8d7\x84\xd2\x83\xb1\xa5\xab\xf2\x1c\xe0\xb0\x04Pj>hE|tr~,^\xb5'
```

Comment:
We absolutely do not want a default key in production.

In `concent-deployment` we will have one but it should be in secrets and not in the repository.